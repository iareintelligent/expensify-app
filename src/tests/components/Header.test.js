import React from "react";
import Header from "../../components/Header";
import { shallow } from "enzyme";

test("should shallow render Header correctly", () => {
    const wrapper = shallow(<Header />);
    // expect(toJson(wrapper)).toMatchSnaupshot(); // not needed bc of "enzyme-to-json/serializer" in jest.config.json
    expect(wrapper).toMatchSnapshot();
    // expect(wrapper.find("h1").text()).toBe("Expensify");
});
