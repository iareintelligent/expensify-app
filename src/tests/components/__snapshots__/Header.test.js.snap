// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should shallow render Header correctly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <nav>
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      className="mr-3"
      exact={true}
      to="/"
    >
      Dashboard
    </NavLink>
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      className="mr-3"
      exact={true}
      to="/create"
    >
      Create Expense
    </NavLink>
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      className="mr-3"
      exact={true}
      to="/help"
    >
      Help
    </NavLink>
  </nav>
</header>
`;
